scripts/install-mender.sh

```diff
@@ -326,7 +326,109 @@ do_install_missing_monitor_dirs () {
     fi
 }
 
+command_exists() {
+    command -v "$@" > /dev/null 2>&1
+}
+
+# Set the LSB_DIST and DIST_VERSION variables guessing the distribution and version;
+# It also checks if this is a forked Linux distro.
+# Credits: https://get.docker.com/
+check_dist_and_version() {
+    # Every system that we officially support has /etc/os-release
+    if [ -r /etc/os-release ]; then
+        LSB_DIST="$(. /etc/os-release && echo "$ID" | tr '[:upper:]' '[:lower:]')"
+    fi
+    case "$LSB_DIST" in
+        ubuntu)
+            if command_exists lsb_release; then
+                DIST_VERSION="$(lsb_release --codename | cut -f2)"
```

Comment:
cut has been known to occasionally change the default -d, could we have explicit -d here?

Reply:
actually the safest is: 
```bash
| awk '{print($NF);}'
```